playbooks/sample-sap-hypervisor-redhat_ocp_virt-preconfigure.yml

```diff
@@ -4,8 +4,9 @@
   serial: 1
   vars:
     sap_hypervisor_node_platform: redhat_ocp_virt
-
-  tasks:
-    - name: Include Role
-      ansible.builtin.include_role:
-        name: sap_hypervisor_node_preconfigure
+    sap_hypervisor_node_kubeconfig: "{{ lookup( 'ansible.builtin.env', 'KUBECONFIG') }}"
+  environment:
+    KUBECONFIG: "{{ sap_hypervisor_node_kubeconfig }}"
+    K8S_AUTH_KUBECONFIG: "{{ sap_hypervisor_node_kubeconfig }}"
+  roles:
+    - { role: community.sap_install.sap_hypervisor_node_preconfigure }
```

Comment:
this is not correct syntax, should be 
```ansible 
  roles:
   - community.sap_install.sap_hypervisor_node_preconfigure
```

Reply:
@kksat Please provide evidence to support your statement. The latest [Using Ansible playbooks -> Working with playbooks -> Roles](https://docs.ansible.com/ansible/latest/playbook_guide/playbooks_reuse_roles.html) document contains the  syntax you rejected in [this section](https://docs.ansible.com/ansible/latest/playbook_guide/playbooks_reuse_roles.html#id12). To me, it appears that this syntax is not deprecated or forbidden and we can safely leave it in place.

Reply:
my mistake, we of course can leave this as it is.